core/src/test/java/com/yahoo/ycsb/generator/TestZipfianGenerator.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2010 Yahoo! Inc. All rights reserved.
```

Comment:
nit: the year here should be the year of authorship. I'm fine fixing this while doing a sanity check on license headers as a part of #266. Let me know if you want to fix it now instead. If I don't hear anything I'll merge this tonight.